sqlglot/dialects/exasol.py

```diff
@@ -564,3 +566,205 @@ def rank_sql(self, expression: exp.Rank) -> str:
             if expression.args.get("expressions"):
                 self.unsupported("Exasol does not support arguments in RANK")
             return self.func("RANK")
+
+        def table_sql(self, expression: exp.Table, sep: str = " AS ") -> str:
+            """
+            If a table has PIVOTs attached, let the Pivot render a derived table .
+            """
+            pivots = expression.args.get("pivots") or []
+            if not pivots:
+                return super().table_sql(expression)
+
+            if len(pivots) > 1:
+                self.unsupported("Multiple PIVOT clauses are not supported by Exasol")
+                return super().table_sql(expression)
+            pivot = pivots[0]
+            return self.sql(pivot)
+
+        def subquery_sql(self, expression: exp.Subquery, sep: str = " AS ") -> str:
+            pivots = expression.args.get("pivots") or []
+            if not pivots:
+                return super().subquery_sql(expression)
+
+            if len(pivots) > 1:
+                self.unsupported("Multiple PIVOT clauses are not supported by Exasol")
+                return super().subquery_sql(expression)
+            pivot = pivots[0]
+            return self.sql(pivot)
+
+        def pivot_sql(self, expression: exp.Pivot) -> str:
+            """
+            Exasol does not support PIVOT, so we rewrite it.
+
+            Rewrite:
+            SELECT ... FROM T PIVOT (...)
+
+            Into:
+            SELECT ... FROM (
+                SELECT <group cols>,
+                        <agg(CASE WHEN ...)>
+                FROM T
+                GROUP BY <group cols>
+            )
```

Comment:
This conversion is easy to mess up if you change the scopes by moving both the pivot and `T` inside of a new derived table. What if you have columns in the original scope already qualified with either `T` or the pivot's alias, if any? Have you thought / handled cases like this?